src/tests/utils/convert.js

```diff
@@ -1,20 +1,31 @@
 require('should');
+const requireFromString = require('require-from-string');
 const convert = require('../../utils/convert');
 const definitions = {
   basic: require('./definitions/basic.json'),
+  basicScripting: require('./definitions/basic-scripting.json'),
   apiHeader: require('./definitions/api-header.json'),
   apiQuery: require('./definitions/api-query.json'),
   session: require('./definitions/session.json'),
   oauth2: require('./definitions/oauth2.json'),
+  oauth2Scripting: require('./definitions/oauth2-scripting.json'),
   oauth2Refresh: require('./definitions/oauth2-refresh.json'),
+  oauth2RefreshScripting: require('./definitions/oauth2-refresh-scripting.json'),
 };
 
 /* eslint no-eval: 0 */
-const s2js = (string) => eval(`
-const AuthTest = { operation: { perform: 'FAKE_PERFORM_FUNCTION' } };
-const getSessionKey = 'FAKE_GET_SESSION_KEY_FUNCTION';
-(${string});
-`);
```

Comment:
Since now I'm using `loadAuthModuleFromString` to eval the generated auth code, we don't need `AuthTest` and `getSessionKey` in `s2js`.